hcl.md: update-hcl-cpu

Stanislaw-bnk opened the pull request.

pietrushnic commented on docs/unified/msi/hcl.md:
```diff
@@ -44,20 +44,15 @@ hardware.
-        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL Report |
-        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL report |
         | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | [Qubes HCL reports][2] |
+        | 12th Gen Intel(R) Core(TM) i7-12700K | v1.0.0 | Dasharo HCL Report |
```
@Stanislaw-bnk, you face these weird, wrong changes because your entries were not sorted. Please note that the line has just moved from two lines above, so there is no change here.

pietrushnic commented on docs/unified/msi/hcl.md:
```diff
@@ -72,6 +67,7 @@ hardware.
         | 13th Gen Intel(R) Core(TM) i5-13600K | v0.9.0 | Dasharo HCL Report |
         | 13th Gen Intel(R) Core(TM) i5-13600K | v1.1.2 | Dasharo HCL Report |
         | 14th Gen Intel(R) Core(TM) i7-14700K | v0.9.0 | Dasharo HCL Report |
+        | Intel(R) Core(TM) i7-14700K | v0.9.0 | Dasharo HCL Report |
```
There is something wrong here; there should be a 14th at the beginning for consistency.

pietrushnic commented on docs/unified/msi/hcl.md:
```diff
@@ -86,11 +82,15 @@ hardware.
+        | Intel(R) Core(TM) i9-14900K | v0.9.0 | Dasharo HCL Report |
+        | Intel(R) Core(TM) i9-14900K | v0.9.1 | Dasharo HCL Report |
+        | Intel(R) Core(TM) i9-14900KF | v0.9.0 | Dasharo HCL Report |
+        | Intel(R) Core(TM) i9-14900KS | v0.9.1 | Dasharo HCL Report |
```
Lack of 14th at the beginning and a lack of sorting.

Stanislaw-bnk commented:
@pietrushnic fixed all the above in the latest commit: https://github.com/Dasharo/docs/pull/964/commits/9c18a93973971c7e39b5cfb96bb7cbdc780242f6
I've added the numbering (14th Gen) for the entries that were missing this information in the domestic report, sorted the list in ascending order, and removed a few duplicates that were found.

Stanislaw-bnk commented:
@pietrushnic ping for review.

pietrushnic reviewed (changes requested):
@Stanislaw-bnk how is this possible that we remove than we adding, this is essentially not possible, unless you didn't check all reports.
